admin/acceptance.py

```diff
@@ -156,23 +127,26 @@ def __init__(self):
             raise UsageError("Unsupored varianta: %s."
                              % (', '.join(self.variants),))
 
-    def start_nodes(self):
+    @inlineCallbacks
```

Comment:
I'm going to say this is replacing synchronous linear code, so using `inlineCallbacks` makes for a smaller, easier to review diff.